File: FashionClub.frontend/fc70/src/models/Dtos/Brands/BrandMixOverviewDto.ts
import { BrandDto } from "./BrandDto";
import { BrandOverviewDto } from "./BrandOverviewDto";
import { CompetitorBrandDto } from "./CompetitorBrandDto";


export interface BrandmixOverviewDto {
    brands?: BrandDto[];
    competitorBrands?: CompetitorBrandDto[];
    brandsOverviews: BrandOverviewDto[];
}

File: FashionClub.Core.Application/DTOs/BrandmixOverviewDTO.cs
using FashionClub.Domain.Models;

namespace FashionClub.Core.Application.DTOs;

public record BrandmixOverviewDTO(List<BrandDTO>? Brands, List<CompetitorBrandDTO>? CompetitorBrands, List<BrandOverviewDTO>? BrandOverviewDto );